src/Overview/Overview.js

```diff
@@ -0,0 +1,29 @@
+import './Overview.scss';
+
+function Overview() {
```

Comment:
We'll have to think about what makes sense to put here, and what data we can easily acquire, but I like having this overview.